src/hdmf/build/manager.py

```diff
@@ -97,6 +98,10 @@ def namespace_catalog(self):
     def type_map(self):
         return self.__type_map
 
+    @property
+    def export(self):
```

Comment:
We should document why ``export`` is a property of the ``BuildManager`` compared to it being just an argument on the ``build`` function. I assume this is the case because it changes state of the manager overall, i.e., a BuildManager with export=true has container sources set in its __containers and with export=false it does not. And export should also not be able to change after the fact for that reason. Is this correct?

Reply:
A `BuildManager` with export=True has no cached builders and ignores existing container source values. A `BuildManager` with export=False checks that the container is being written to is the same as the container source when it exists.

The `export` field does not change after initialization.

I added the requested documentation